manual/manual/library/Makefile

```diff
@@ -90,7 +95,7 @@ $(INTF): $(MLIS)
 	mv -f Ast_mapper.tex Astmapper.tex
 
 Tk.tex: tk.mli
-	$(CSLDIR)/byterun/ocamlrun $(CSLDIR)/ocamldoc/ocamldoc -latex \
+	$(OCAMLDOC) -latex \
```

Comment:
I am not sure if it makes sense to update this part of the Makefile which is a forgotten remnant of labltk.